src/web/mjs/connectors/AniGliScans.mjs

```diff
@@ -0,0 +1,26 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
```

Comment:
This has already been merged, remove aniglitch stuff

Reply:
Sorry, Should I remove the connector and send a PR?

Reply:
Remove it from the branch you submitted the PR. Your master.